### DISCUSSION TITLE
Why can't I enable minestorm dependency?

### DISCUSSION THREAD

**Discussion by miieev** [Minestom/Minestom#2310] [Help] [answered]
Hello everyone, why can't I connect minestorm dependency?
![image](https://github.com/user-attachments/assets/c1f4d77b-93c4-4e9b-9bb0-9f81ebcb2a65)

**Comment by cosrnic** [comment]
you need to use java 21 for minestom

**Comment by mudkipdev** [comment, marked as answer]
Set jvmToolchain to 21

**Comment by miieev** [comment]
Oh...
Thanks

**Reply by mudkipdev** [reply]
Please mark my comment as correct answer
